From 03effc15d64618f86a6586ff73f33a0c4a591012 Mon Sep 17 00:00:00 2001
From: Willy Tarreau <w@1wt.eu>
Date: Wed, 12 Feb 2014 14:55:41 +0100
Subject: [PATCH 16/17] MINOR: ssl: add DEFAULT_SSL_MAX_RECORD to set the
 record size at build time

For some deployments it may help to have tune.ssl.maxrecord set to a
more efficient one at build time. This build setting allows this.
---
 src/haproxy.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/haproxy.c b/src/haproxy.c
index 4bd30a1..5ba7a73 100644
--- a/src/haproxy.c
+++ b/src/haproxy.c
@@ -142,6 +142,9 @@ struct global global = {
 		.chksize = BUFSIZE,
 #ifdef USE_OPENSSL
 		.sslcachesize = SSLCACHESIZE,
+#ifdef DEFAULT_SSL_MAX_RECORD
+		.ssl_max_record = DEFAULT_SSL_MAX_RECORD,
+#endif
 #endif
 #ifdef USE_ZLIB
 		.zlibmemlevel = 8,
-- 
1.8.3.2

